plane-detection.bs

```diff
@@ -262,6 +320,7 @@ In order to <dfn>update plane object</dfn> |plane| from a [=native plane object=
     1. Set |plane|'s [=XRPlane/frame=] to |frame|.
     1. If |native plane| is classified by the underlying system as vertical, set |plane|'s {{XRPlane/orientation}} to {{XRPlaneOrientation/"vertical"}}. Otherwise, if |native plane| is classified by the underlying system as horizontal, set |plane|'s {{XRPlane/orientation}} to {{XRPlaneOrientation/"horizontal"}}. Otherwise, set |plane|'s {{XRPlane/orientation}} to <code>null</code>.
     1. Set |plane|'s {{XRPlane/polygon}} to the new array of vertices representing |native plane|'s polygon, performing all necessary conversions to account for differences in native plane polygon representation.
+    1. <div class="unstable">Set |plane|'s {{XRPlane/semanticLabels}} to a new sequence of [=semantic label|semantic labels=].</div>
```

Comment:
Nit: how should this sequence be populated? We should probably elaborate, sth like "new sequence of semantic labels that match the underlying XR system's understanding of what the planes represent".

We may also want to elaborate on what should happen if multiple labels can be applied. Maybe stupid example: how will a wall containing doors & windows be labeled? How about category labels? If we ever introduce "furniture" label, do we guarantee the order of items in this sequence in any way (e.g. desk is labeled `["furniture", "desk"]`?).

Reply:
I agree. I made this reflect what is done in OpenXR but I believe we never put more than 1 label in the array.

Reply:
update change to list just 1 label